src/drivers/serial/rpi_mini_uart/rpi_mini_uart.c

```diff
@@ -0,0 +1,135 @@
+/**
+ * @file
+ * @brief Raspberry Pi 16550-like Mini UART driver
+ * Mini UART is the default console device on Raspberry Pi models with
```

Comment:
Is this OK? Any format issue?
I added more info about the other registers which are different with the ones in NS16550.

In fact, the PR message is the one I added in the commit message.

Reply:
Ok. Thank you very much. It's enough!